src/stores/views.ts

```diff
@@ -136,7 +136,7 @@ export function viewSetSelectedTimeline(timelineId: number | null): void {
     const firstRow = currentTimeline.rows[0];
 
     if (firstRow) {
-      viewSetSelectedRow(firstRow.id);
+      // viewSetSelectedRow(firstRow.id);
```

Comment:
Can we just remove this condition since it's not used?

Reply:
Yeah, might break the Timeline Form if we're ok with that?